map-of-the-web/motw/motw/producing.py

```diff
@@ -0,0 +1,132 @@
+from bokeh.plotting import figure, output_file, show
+from bokeh.io import save
+from bokeh.layouts import gridplot 
+from motw.snapshot import Snapshot
+import requests as req
+from datetime import datetime
+import pandas as pd
+
+def timestamp2date(x):
+	return x[0:4]+"-"+x[4:6]+"-01"
+
+def Frequency(snapshot_list):
```

Comment:
Could you add some brief description about functions and objects in the form of a comment?  This will help me and others understand the purpose.

I may be misunderstanding, but I believe this is a function, not a class definition.  If that is the case [it should be lower case](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) <-- check out that style guide.